transports/uds/src/lib.rs

```diff
@@ -164,14 +163,15 @@ fn multiaddr_to_path(addr: &Multiaddr) -> Result<PathBuf, ()> {
 
 #[cfg(test)]
 mod tests {
+    extern crate tokio;
+    use self::tokio::runtime::current_thread::Runtime;
```

Comment:
Is `self` needed here?

Reply:
I could use tests instead of self but it fails resolve without either.